packages/dashboard-hardhat-plugin/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@truffle/dashboard-hardhat-plugin",
+  "version": "0.1.0-0",
+  "description": "Truffle Dashboard for Hardhat plugin",
+  "repository": "github:trufflesuite/dashboard-hardhat-plugin",
+  "author": "ConsenSys Software Inc",
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "hardhat-plugin",
+    "truffle",
+    "truffle-dashboard"
+  ],
+  "scripts": {
+    "lint:fix": "prettier --write 'src/**/*.{js,ts}' 'test/**/*.{js,ts}' && tslint --fix --config tslint.json --project tsconfig.json",
+    "lint": "tslint --config tslint.json --project tsconfig.json",
```

Comment:
is this lint stuff necessary? isn't this included as part of normal monorepo stuff? @cds-amal?

Reply:
I expect husky to lint at commit time. Was there a husky issue @kevinbluer?